services/cd-service/pkg/service/tags.go

```diff
@@ -0,0 +1,44 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/freiheit-com/kuberpult/pkg/api"
+	"github.com/freiheit-com/kuberpult/services/cd-service/pkg/repository"
+)
+
+type TagsServer struct {
+	Cfg repository.RepositoryConfig
+}
+
+func (s *TagsServer) GetGitTags(ctx context.Context, in *api.GetGitTagsRequest) (*api.GetGitTagsResponse, error) {
+	tags, commits, err := repository.GetTags(s.Cfg, "./repository", ctx)
```

Comment:
I would prefer if this endpoint has no influence on other endpoints operating on `./repository`:
```suggestion
	tags, commits, err := repository.GetTags(s.Cfg, "./repository_tags", ctx)
```